packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -32,11 +49,12 @@ contract Bridge is EssentialContract, IBridge {
     uint256 private constant _SEND_ETHER_GAS_LIMIT = 35_000;
 
     /// @dev Place holder value when not using transient storage
-    uint256 internal constant PLACEHOLDER = type(uint256).max;
+    uint256 private constant _PLACEHOLDER = type(uint256).max;
 
     /// @notice The next message ID.
     /// @dev Slot 1.
-    uint128 public nextMessageId;
+    uint64 private __reserved1;
+    uint64 public nextMessageId;
```

Comment:
Is there any particular reason using `uint64` over `uint128` ?
As `nextMessageId` is used in `struct Message: id`, and `message` is either `calldata` or `memory`, there won't be any optimization gained from data-packing (bc. there is no data packing there).

Memory layout spec:
<img width="823" alt="kép" src="https://github.com/taikoxyz/taiko-mono/assets/51912515/6ca32eae-3712-464d-a350-b5411abc0e18">


Calldata layout spec:
<img width="766" alt="kép" src="https://github.com/taikoxyz/taiko-mono/assets/51912515/3c37b051-3cef-4924-b1a2-d0060833b46e">

Link: https://docs.soliditylang.org/en/latest/internals/layout_in_memory.html

Reply:
If we don't consider the dynamic-size bytes in Message, the previous Message definition uses 320bytes, the new definition uses 192 bytes. he padding to 32 bytes doesn't apply to each field.

Reply:
> If you have a struct with 2 uint128 fields, it will be packed into a 32 bytes, not 64 bytes.

Only in case of storage there is data packing savings.
![kép](https://github.com/taikoxyz/taiko-mono/assets/51912515/5c288a23-84a2-44fd-b881-1394e25c0b6b)

 I did not comment because i think there is an issue here, just wanted to say, that we dont optimize here anything, because `message` (which has the `id` field  as `nextMessageId`)struct (is memory or calldata) so data packing will not be done. 
 
 But anyway, we also dont lose anything here, so all good!